src/main/java/org/recommendation/data/UserDataInitilizer.java

```diff
@@ -0,0 +1,84 @@
+package org.recommendation.data;
+
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.xssf.usermodel.XSSFSheet;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.recommendation.data.helper.UserDataHelper;
+import org.recommendation.log.Logger;
+import org.recommendation.log.SoutLogger;
+import org.recommendation.model.User;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.TreeMap;
+
+
+public class UserDataInitilizer extends DataInitializer {
+    private final Logger logger = new SoutLogger();
+    private Map<String, User> userMap = new HashMap<>();
+
+
+    @Override
+    public Map<String, String[]> readAndCleanData(final BufferedReader br) {
+        Map<String, String[]> dataStore = new TreeMap<>();
+        String line;
+        try {
+            Integer counter = 0;
+            while ((line = br.readLine()) != null) {
+                String[] data = line.split("\\s+");
+                int totalItems = data.length;
+                if (totalItems > 1) {
```

Comment:
what if totalItem is not greater than 1 ? Do we need handling for the same?

Reply:
if total item is <1 means that data is corrupted or insufficient, as discussed previously we are silently bypassing those cases